packages/@webex/plugin-meetings/src/multistream/mediaRequestManager.ts

```diff
@@ -122,17 +129,15 @@ export class MediaRequestManager {
     // reduce max-fs until total macroblocks is below limit
     for (let i = 0; i < maxFsLimits.length; i += 1) {
       let totalMacroblocksRequested = 0;
-      Object.entries(clientRequests).forEach(([id, mr]) => {
+      Object.values(clientRequests).forEach((mr) => {
         if (mr.codecInfo) {
           mr.codecInfo.maxFs = Math.min(
             mr.preferredMaxFs || CODEC_DEFAULTS.h264.maxFs,
             mr.codecInfo.maxFs || CODEC_DEFAULTS.h264.maxFs,
             maxFsLimits[i]
           );
           // we only consider sources with "live" state
-          const slotsWithLiveSource = this.clientRequests[id].receiveSlots.filter(
```

Comment:
the old code here was using `this.clientRequests` instead of the cloned `clientRequests` as a workaround to the "attempted to get private field on non-instance" error that we would get when accessing `ReceiveSlot.sourceState` (which is a private field, so it doesn't get cloned). Now I needed to access the cloned requests, because they may have been trimmed, so I needed to fix this issue by cloning the client requests properly - see `cloneClientRequests()`